2025/linux/solutions.md/task6.md

```
@@ -0,0 +1,80 @@
+# AUTOMATE BACKUPS WITH  SHELL SCRIPTING 
+
+In general , shell scripts configure which directories to backups, and passes those directories as arguments to tar utility, which creates an archive file.the archived file can be moved or copied to another location, or can be created remotely on the remote file system or network file system(NFS).
+
+The tar utility creates one archive file out of many files or directories. tar can also filter the files through compression utilities, thus reducing the size of the archive file.
+
+!/bin/bash
+####################################
+#
+# Backup to NFS mount script.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)